src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -176,6 +209,91 @@ function removeRubricCol(index, questionNum) {
                                          StatusType.WARNING);
 }
 
+function swapRubricCol(questionNum, colIndex, isSwapLeft) {
+    if ($(`#rubricEditTable-${questionNum}`).length === 0
+            && $(`.rubricCol-${questionNum}-${colIndex}`).length === 0
+            && typeof isSwapLeft !== 'boolean') {
```

Comment:
I probably missed this earlier, but looking at it again, should the `&&` be replaced by `||`? It appears that this particular conditional checks for failing conditions, in which case having any of them be true should be sufficient to do an early return.